r/rsparkling/man/as_h2o_frame.Rd

```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2: do not edit by hand
+% Generated by roxygen2 (4.1.1): do not edit by hand
```

Comment:
Docs need to be updated since we added an extra parameter to `h2o_flow()`. I have re-updated the docs with the latest version of roxygen2.